zone/raids.h

```diff
@@ -259,6 +295,9 @@ class Raid : public GroupIDConsumer {
 
 	RaidMember members[MAX_RAID_MEMBERS];
 	char leadername[64];
+	char MainAssisterPCs[MAX_RAID_MAIN_ASSISTERS][64];
+	char MainMarkerPCs[MAX_RAID_MAIN_MARKERS][64];
```

Comment:
This needs to be `snake_case` eg `main_marker_pcs`

Reply:
Updated